From 0e6448b5c2115a90314143ddde3a00196b2f24fe Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Fri, 27 Sep 2024 11:14:43 +0800
Subject: [PATCH 1/3] fix(CMake/Documentation.cmake): fix XSL stylesheets
 include path

AOSC OS does not ship the unversioned /usr/share/xml/docbook/xsl-stylesheets,
but one with a versioned suffix.

This may be fixed in the future (with a symlink).
---
 CMake/Documentation.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMake/Documentation.cmake b/CMake/Documentation.cmake
index f3eb4c5e1..52ba8a454 100644
--- a/CMake/Documentation.cmake
+++ b/CMake/Documentation.cmake
@@ -31,7 +31,7 @@ find_path(DOCBOOK_XSL manpages/docbook.xsl
          # OpenSUSE
          /usr/share/xml/docbook/stylesheet/nwalsh/current
          # Arch
-          /usr/share/xml/docbook/xsl-stylesheets
+          /usr/share/xml/docbook/xsl-stylesheets-1.79.2
          # Fedora
          /usr/share/sgml/docbook/xsl-stylesheets
          # Fink
-- 
2.46.2

